frontend/src/components/Dropdown/index.ts

```diff
@@ -0,0 +1,77 @@
+import { createElement } from "@/utils/dom";
+import "./style.scss";
+
+export interface IDropdownItem {
+  text: string;
+  danger?: boolean;
+  className?: string;
+  id?: string;
+  onClick?: (event: any) => void;
+}
+
+export interface IDropdownProps {
+  style: {
+    [key: string]: string;
+  };
+  items: IDropdownItem[];
+  delegate?: (event: any) => void;
+  parentSelector: string;
+}
+
+const template = (props: IDropdownProps): string => `
+  <div class="dropdown" style="${makeStyle(props.style)}">
+    ${props.items.map((item: IDropdownItem) => makeDropdownItem(item)).join("")}
```

Comment:
```suggestion
    ${props.items.map(makeDropdownItem).join("")}
```

이런 식으로 표현도 가능하겠네요.

`makeDropdownItem`을 컴포넌트화 해서 상단으로 올리거나 분리하는 건 어떨까요?

Reply:
상단으로 올려보도록 하겠습니다 !!